spec/system/add_search_spec.rb

```diff
@@ -18,7 +18,7 @@
     @ruby_devloper_english = create :developer,
                                     email: 'quang.pham@example.com',
                                     programming_languages: [ruby_programming_language],
-                                    languages: [@english]
+                                    languages: [english]
```

Comment:
If you refer to `english` only here, you can write `languages: [create(:language, code: 'en')]`.
With this, other developers don't need to read extra code.
But the original code is totally fine.

Reply:
@jp-ryuji 👍